application/features/VNC.py

```diff
@@ -65,8 +68,10 @@ def connect(self, *args, **kwargs):
         return super().connect(*args, **kwargs)
 
     def get_vnc_password(self):
-        _, _, stdout, _ = self.exec_command_blocking('xxd -p ~/.vnc/passwd')
-        hexdump = stdout.readline()
+        # return True, "123456"
+        _, _, stdout, _ = self.exec_command_blocking("hexdump --format '16/1 \"%02x\"' ~/.vnc/passwd")
+        hexdump = stdout.readline().rstrip()
+        logger.info(f"hexdump = {hexdump}")
```

Comment:
Probably not a good practice to log password related information (especially when it can be used to deduct the plain text password )
```suggestion
```